src/app/workflow/workflow.component.html

```diff
@@ -46,9 +46,14 @@ <h2 class="text-center">The workflow linked to could not be found!</h2>
     <ngx-json-ld [json]="schema"></ngx-json-ld>
     <div fxFlex="80">
       <h3 id="workflow-path" data-cy="workflowTitle" fxLayout="row" fxLayoutAlign="start center" fxLayoutGap="1rem">
-        <span>
+        <span *ngIf="entryType === EntryType.BioWorkflow || entryType === EntryType.Service; else gitHubAppTool">
```

Comment:
I suggest an explicit check rather than relying on an else. If another type ever gets added, we don't want it to default to a tool icon.